jetty9/src/main/docker/Dockerfile

```diff
@@ -39,8 +39,8 @@ RUN mkdir webapps $TMPDIR \
  && chown -R jetty:jetty $JETTY_BASE $TMPDIR
 
 # Start Jetty directly
-COPY docker-entrypoint.bash /
-RUN chmod +rx /docker-entrypoint.bash
+COPY app-env.bash /
+RUN chmod +rx /app-env.bash
 
```

Comment:
How about `RUN printf "\nsource /app-env.bash\n" >> /setup-env.bash`, and no modifications to the openjdk runtime?

Also, maybe rename `app-env.bash` -> `setup-env-ext.bash` in that case?